src/applications/appeals/996/constants.js

```diff
@@ -79,6 +79,19 @@ const supportedBenefitTypes = [
   // 'nca',
 ];
 
+/**
+ **** MAX LENGTH ****
+ */
+
+// Values from Lighthouse maintained schema v2
+// see https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/v2/200996.json
+export const HLR_MAX_LENGTH = {
+  HLR_SELECTIONS: 100, // submitted issues (not in schema)
```

Comment:
Remove this. Across all 3 apps, we only allow a max of 100 issues to be submitted. It should be in the schema, but it does appear to be missing for HLR 🤔 Maybe we should tell Lighthouse about this inconsistency

<details><summary>Included issues schema</summary>

<!-- leave a blank line above -->
<img width="404" alt="Screenshot 2023-08-24 at 9 05 45 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/88734d48-44bc-46d7-bdd1-20c4a73617b2">
<img width="423" alt="Screenshot 2023-08-24 at 9 06 22 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/4aa25d3c-f3f0-461a-8c7b-ab0c32612214">
<img width="414" alt="Screenshot 2023-08-24 at 9 07 02 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/722b3cb6-a701-46c6-b06f-cda5711d1b64"></details>